commands/arguments/arguments.go

```diff
@@ -0,0 +1,87 @@
+package arguments
```

Comment:
Since this isn't a command itself, the package should not exist in the top-level commands hierarchy.  I'd put it in `commands/internal/arguments` (the `internal` directory has special meaning, see: https://blog.learngoprogramming.com/special-packages-and-directories-in-go-1d6295690a6b), which allows other command packages to import it and share the code but doesn't make it globally exported.

Reply:
In addition, please add a docstring comment of the syntax `// Package arguments ...` in order to appease go-lint.  A simple line or two describing the purpose of the package will suffice.

Reply:
Will do, thanks. I’m new to go so wasn’t sure of the best way of packaging it

Reply:
This is done, although I wasn't able to get go-lint to produce a warning if that comment was missing, so not totally sure I've fixed it correctly